plugins/woocommerce/client/legacy/js/admin/meta-boxes-order.js

```diff
@@ -1591,6 +1591,89 @@ jQuery( function ( $ ) {
 					return settings;
 				}
 			});
+
+			$( '#order_custom #metakeyselect').filter( ':not(.enhanced)' ).each( function() {
```

Comment:
Nice work getting this up and running!  Been a minute since I've worked on any jQuery 🙉